### DISCUSSION TITLE
[bug-security] Creating a token with privileges higher than user has

### DISCUSSION THREAD

**Discussion by syatihoko** [getsentry/support#43] [Support] [answered]
Sentry 21.11.
The user can create an elevated token. We Checked the user in the Members group creates a token with all checkboxes and renames the project using API.
Maybe I can disable the creation of tokens?

**Comment by mdtro** [comment, marked as answer]
@syatihoko -- Thank you for reporting this! We're looking in to this now.

**Reply by syatihoko** [reply]
Hello @mdtro . Thank you. Please tell me if you managed to confirm the issue?
If it's confirmed. Are there any recommendations or plans to fix it?

**Reply by mdtro** [reply]
Hi @syatihoko! Sorry for the late reply. I was able to re-create your issue and am working on a fix.

**Reply by dashed** [reply]
@syatihoko Just a heads up, the fix is here https://github.com/getsentry/sentry/pull/35845

**Reply by mdtro** [reply]
@syatihoko This has now been resolved! Thank you for bringing it to our attention!
